src/pages/Cart/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Navbar } from '../../components/Home/Navbar/index'
+import { Footer } from '../../components/Home/Footer/index'
+import { HorizontalCard } from '../../components/Home/common/horizontal-card'
+import { CartNavbar} from "../../components/Home/common/cart-navbar"
```

Comment:
Suggestion: You can make a common index file for the imports and then use it in your index file